peps/pep-0755.rst

```diff
@@ -0,0 +1,907 @@
+PEP: 755
```

Comment:
PEP 747 is the next available, please can you use that?

Reply:
Will adjust.